src/library-authoring/library-detail/index.js

```diff
@@ -1,3 +1,2 @@
-export { default as LibraryPage } from './LibraryPage';
 export { default as LibraryAuthoringPage } from './LibraryAuthoringPage';
```

Comment:
The `library-detail` and `edit-library` folder names have become pretty counterintuitive -- `library-detail` holds the page with the "Edit" buttons and `edit-library` is under "Details".

Would a renaming such as this seem intuitive to you?
* `library-detail` -> `author-library`
* `edit-library` -> `configure-library`

If you agree and think it'd be quick, go ahead and do it in this PR. If you disagree or think it'd be too time consuming to be ready for a merge tomorrow, then nevermind; we'll have some time for refactoring in M3 :)

Reply:
Done in my latest push.